src-django/api/models.py

```diff
@@ -97,3 +97,70 @@ def save(self, **kwargs):
 
     class Meta:
         app_label = 'api'
+
+
+class Condition(models.Model):
+    CRITERIA_TYPES = (
+        ('EQUALS', 'EQUALS'),
+        ('GREATER', 'GREATER'),
+        ('LESSER', 'LESSER')
+    )
+
+    NODE_TYPES = (
+        ('AND', 'AND'),
+        ('OR', 'OR'),
+        ('NOT', 'NOT'),
+        ('CRITERIA', 'CRITERIA')
+    )
+
+    parent = models.ForeignKey('self', on_delete=models.CASCADE, related_name='children', blank=True, null=True)
+    element = models.ForeignKey(Element, blank=True, null=True)
```

Comment:
(not sure if you're replying to my comment, or making an unrelated comment about this line :stuck_out_tongue:)